code/game/gamemodes/objective_items.dm

```diff
@@ -96,14 +98,73 @@
 		return TRUE
 	return FALSE
 
-/datum/objective_item/steal/low_risk/cargo_budget
+/datum/objective_item/steal/traitor/det_revolver
+	name = "detective's revolver"
+	targetitem = /obj/item/gun/ballistic/revolver/c38/detective
+	excludefromjob = list(JOB_DETECTIVE)
+	exists_on_map = TRUE
+
+/obj/item/gun/ballistic/revolver/c38/detective/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/gun/ballistic/revolver/c38/detective)
+
+/datum/objective_item/steal/traitor/chief_engineer_belt
+	name = "the chief engineer's belt"
+	targetitem = /obj/item/storage/belt/utility/chief
+	excludefromjob = list(JOB_CHIEF_ENGINEER)
+	exists_on_map = TRUE
+
+/obj/item/storage/belt/utility/chief/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/storage/belt/utility/chief)
+
+/datum/objective_item/steal/traitor/telebaton
+	name = "a head of staff's telescopic baton"
+	targetitem = /obj/item/melee/baton/telescopic
+	excludefromjob = list(
+		JOB_RESEARCH_DIRECTOR,
+		JOB_CAPTAIN,
+		JOB_HEAD_OF_SECURITY,
+		JOB_HEAD_OF_PERSONNEL,
+		JOB_CHIEF_ENGINEER,
+		JOB_CHIEF_MEDICAL_OFFICER
+	)
+	exists_on_map = TRUE
+
+/obj/item/melee/baton/telescopic/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/melee/baton/telescopic)
+
+/datum/objective_item/steal/traitor/cargo_budget
 	name = "cargo's departmental budget"
 	targetitem = /obj/item/card/id/departmental_budget/car
 	excludefromjob = list(JOB_QUARTERMASTER, JOB_CARGO_TECHNICIAN)
 	exists_on_map = TRUE
 
 /obj/item/card/id/departmental_budget/car/add_stealing_item_objective()
-	ADD_STEAL_ITEM(src, /obj/item/card/id/departmental_budget/car)
+	return add_item_to_steal(src, /obj/item/card/id/departmental_budget/car)
+
+/datum/objective_item/steal/traitor/captain_modsuit
+	name = "the captain's magnate MOD control unit"
+	targetitem = /obj/item/mod/control/pre_equipped/magnate
+	excludefromjob = list(JOB_CAPTAIN)
+	exists_on_map = TRUE
+
+/obj/item/mod/control/pre_equipped/magnate/add_stealing_item_objective()
+	return add_item_to_steal(src, /obj/item/mod/control/pre_equipped/magnate)
+
+/datum/objective_item/steal/traitor/captain_spare
+	name = "the captain's spare ID"
```

Comment:
That's fine, interrogating and torturing the captain for the codes seems like a good roleplay avenue. It's for players who think they can take that risk and believe that they can get the spare using any means necessary.

Also, you're slightly incorrect. The captain gets the spare ID safe code combination as a piece of paper.

Reply:
To be fair, you can destroy the locker with acid, it will drop the ID, and there is a hint about it in the description. Still, a cup of fluorosulphic acid barely made a dent in it, it removed like, 8 integrity out of 300, so this would take an unreasonably long time.
Similarly captain spawns with a sheet of paper with the code, but they will probably shred it immediately.

Reply:
ah, you're right about that last part. I misread how that system handled it. My bad. It adds a memory AND a fluff paper.

I guess this might not be entirely mechanically impossible, but I still believe it is uncharacteristically lacking mechanical alternatives due to being held back by a bug (if my talk with Timber is being remembered correctly, hopefully he can chime in there). I can see a player getting frustrated and unable to understand what is being asked of them or how to get the code at all.

Maybe, as a way to help players not walk into this objective unaware of how hard it could be, there should be a more mechanical method for obtaining those codes _from the captain_. One that a player can have explained to them clearly. Like, for instance, a surgery that lets you extract memories that someone who takes this objective can use by a provided surgery disk. Anything to make this less steep of an expectation of roleplay and actually provides a mechanical avenue when roleplay won't work (this is actually a very reasonable expectation).

Reply:
> To be fair, you can destroy the locker with acid, it will drop the ID, and there is a hint about it in the description. Still, a cup of fluorosulphic acid barely made a dent in it, it removed like, 8 integrity out of 300, so this would take an unreasonably long time. Similarly captain spawns with a sheet of paper with the code, but they will probably shred it immediately.

Oh man if it got fixed in the intervening weeks I talked to Timber about this, then don't even worry about it, I'll just update the description on the safe.

Reply:
i think the issue is that we are too paranoid about the captain's spare going out that we stunted the sandbox to make sure nobody ever gets it, even though without the ability to copy its access the protection isn't completely justified